conf/sattune.yaml

```diff
@@ -0,0 +1,3 @@
+---
+satellite_physical_disk: /dev/sda 
```

Comment:
Can we autodetect this? In lots of cases this will be on LVM and mounted from somewhere else.

Reply:
Hmm, that would be probably too hard to do it right. What about simple role which would change the values + documentation for that role + having it commented out in https://github.com/redhat-performance/satellite-tune/blob/master/ansible/postgresql.yaml by default?

Reply:
@jhutar I agree, this will be a much better approach, leaving the control in the hand of user, if they want to run this detection feature or not.
Taking about autodetection, though it is a bit harder approach, but we can indeed autodetect this